lib/portlayer/storage/vsphere/parent.go

```diff
@@ -48,29 +42,20 @@ const parentMFile = "parentMap"
 
 // Implements the cache used to lookup an image's parent
 type parentM struct {
-	// location in the datastore in datastore URI format
-	mFilePath string
-
 	// map of image ID to parent ID
 	db map[string]string
 
-	sess *session.Session
+	// roots where the map is stored
+	ds *datastore
 
 	l sync.Mutex
 }
 
 // Starts here.  Tries to create a new parentM or load an existing one.
-func restoreParentMap(ctx context.Context, s *session.Session) (*parentM, error) {
-
-	// Use the UUID of VCH as the root in the datastore.
-	host, err := guest.UUID()
-	if err != nil {
-		return nil, err
-	}
-
+// basepath is the directory path (not in url form) to root the map
```

Comment:
Stale comment.  It used to be a param, but I found later that I didn't need it since we're passing in a *datastore to this.  It itself has a `rootdir` so `basepath` isn't necessary.  Will remove.